docs: point dev checks to just

voctory opened the pull request:
Update install and contributing guides to use the root justfile helpers (`just fmt`, `just fix -p <crate>`, and targeted tests) instead of the older cargo fmt/clippy/test instructions that have been in place since 459363e17b. This matches the justfile relocation to the repo root in 952d6c946 and the current lint/test workflow for CI (see `.github/workflows/rust-ci.yml`).

voctory committed:
docs: point dev checks to just

Update install and contributing guides to use the root justfile helpers (`just fmt`, `just fix -p <crate>`, and targeted tests) instead of the older cargo fmt/clippy/test instructions that have been in place since 459363e17b. This matches the justfile relocation to the repo root in 952d6c946 and the current lint/test workflow.

etraut-openai commented:
@voctory, thanks for the contribution. Please take a look at the codex code review feedback above.

voctory committed:
docs: note just/nextest prerequisites

Address review feedback by adding install steps for just (used by the root justfile) and cargo-nextest (used by just test) in docs/install.md, with a cargo test --all-features fallback for environments without nextest.

voctory commented:
@codex review

voctory commented:
@etraut-openai Thanks for taking a look - addressed the feedback and passes the codex review.

etraut-openai commented:
Thanks for the contribution! Merged.
